### DISCUSSION TITLE
Draw a texture fiber at 45 degree to ND

### DISCUSSION THREAD

**Discussion by AdamLeaghj** [mtex-toolbox/mtex#1739] [Ask Anything] [answered]
Hi everyone,

My intent is to draw a texture fiber with its orientation tilted 45 degrees to the ND axis in {112} pole figure. The illustration of the lattice rotation seems like the below image. I know the result in reality but I'm trying to use Matlab to plot it.  I tried to use fiber texture in one of the tutorials but seems like it didn't work well. Any suggestion on this would be very much appreciated. Thank you.

Regards,
Adam.

**Comment by lfgmorales** [comment]
Hi

Can you expand why it didn’t work well, and paste your script (and figures) to see that the problem is?

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 17 Feb 2023, at 17:53, AdamLeaghj ***@***.***> wrote:

﻿

Hi everyone,

My intent is to draw a texture fiber with its orientation tilted 45 degrees to the ND axis in {112} pole figure. The illustration of the lattice rotation seems like the below image. I know the result in reality but I'm trying to use Matlab to plot it. I tried to use fiber texture in one of the tutorials but seems like it didn't work well. Any suggestion on this would be very much appreciated. Thank you.

[image]<https://user-images.githubusercontent.com/118141541/219714064-23a807da-360f-4129-a508-9fd1db591c2b.png>

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1739>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN67HOI7GS77EBSIU2LWX6UHRANCNFSM6AAAAAAU7UVO5Q>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1739", "url": "https://github.com/mtex-toolbox/mtex/discussions/1739", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by AdamLeaghj** [reply]
Hi,
Please see my code used and the generated plot below. Thanks,

Adam.

```
cs = crystalSymmetry('m-3m');
ori_1 = orientation.byMiller([1 1 2],[1 1 2],cs);
plotPDF(ori_1,Miller({1,1,2},cs),'linewidth',3,'lineColor','orange')
```
![image](https://user-images.githubusercontent.com/118141541/219724042-8295706b-8e97-44ff-9dcb-25a0f8b53d37.png)

**Comment by lfgmorales** [comment, marked as answer]
Hi David,

I think first you have to generate a fiber texture, something like

<https://mtex-toolbox.github.io/FibreODFs.html>
Fibre ODFs | MTEX<https://mtex-toolbox.github.io/FibreODFs.html>
mtex-toolbox.github.io<https://mtex-toolbox.github.io/FibreODFs.html>
        [favicon.png] <https://mtex-toolbox.github.io/FibreODFs.html>


Then rotate 45 degrees into x, y or z and plot it as pole figure?

Cheers

Luiz


Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 17 Feb 2023, at 18:30, AdamLeaghj ***@***.***> wrote:

﻿

Hi,
Please see my code used and the generated plot below. Thanks,

David.

cs = crystalSymmetry('m-3m');
ori_1 = orientation.byMiller([1 1 2],[1 1 2],cs);
plotPDF(ori_1,Miller({1,1,2},cs),'linewidth',3,'lineColor','orange')


[image]<https://user-images.githubusercontent.com/118141541/219724042-8295706b-8e97-44ff-9dcb-25a0f8b53d37.png>

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1739#discussioncomment-5012833>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5INY5365DPJBACCC2K3TWX6YRRANCNFSM6AAAAAAU7UVO5Q>.
You are receiving this because you commented.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1739#discussioncomment-5012833", "url": "https://github.com/mtex-toolbox/mtex/discussions/1739#discussioncomment-5012833", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by AdamLeaghj** [reply]
Thanks Luiz, I finally figured it out after a few days. Thanks for the help.

**Comment by Liujianhua00** [comment]
As to my knowledge, a fiber is defined by two orientations. Firstly, I think you should know the two orientations related to your need. Secondly, the fiber can be plotted in the Euler space, pole figures and inverse pole figures. which way do you need?

**Reply by AdamLeaghj** [reply]
Thanks
